packages/labs/analyzer/src/test/analyzer_test.ts

```diff
@@ -4,36 +4,71 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-import {test} from 'uvu';
+import {suite} from 'uvu';
 // eslint-disable-next-line import/extensions
 import * as assert from 'uvu/assert';
 import ts from 'typescript';
 import * as path from 'path';
 import {fileURLToPath} from 'url';
 
 import {Analyzer} from '../lib/analyzer.js';
+import {AbsolutePath} from '../lib/paths.js';
 
-test('Basic elements - isLitElement', () => {
-  const basicElementsPackagePath = fileURLToPath(
-    new URL('../test-files/basic-elements', import.meta.url).href
-  );
-  const analyzer = new Analyzer(basicElementsPackagePath);
+const basicTest = suite('Basic');
+
+const basicElementsPackagePath = fileURLToPath(
+  new URL('../test-files/basic-elements', import.meta.url).href
+) as AbsolutePath;
+
+let analyzer: Analyzer;
+
+basicTest.before(() => {
+  analyzer = new Analyzer(basicElementsPackagePath);
+});
+
+basicTest('Reads project files', () => {
```

Comment:
You can scope data to a suite in uvu by giving types to the context object, like so:

```suggestion
const basicTest = suite<{analyzer: Analyzer, package: AbsolutePath}>('Basic');

basicTest.before((ctx) => {
  ctx.package = const basicElementsPackagePath = fileURLToPath(
  new URL('../test-files/basic-elements', import.meta.url).href
) as AbsolutePath;
  ctx.analyzer = 
  analyzer = new Analyzer(basicElementsPackagePath);
});

basicTest('Reads project files', ({analyzer, package}) => {
```

Reply:
I'm a little unsure if this delivers a lot, but I'll try it!